core/src/main/resources/AppConstants.properties

```diff
@@ -287,7 +286,7 @@ ldap.auth.dataadmin.base=
 manageDatabase.active=true
 manageDatabase.webServiceListener.active=false
 checkReload.active=true
-ibisDoc.active=true
+ibisConfigXsd.active=true
```

Comment:
Ik zou de property een iets duidelijkere naam willen geven, generateIbisConfigXSD.active of zo iets. En deze standaard uit zetten en alleen in in stage LOC deze aan zetten.